web/themes/custom/liibarrio2020/scss/layout/_hero.scss

```diff
@@ -16,7 +16,7 @@
   }
 
   &__content {
-    height: 0; //height value needed for IE11
+    height: 100; //height value needed for IE11
     min-height: 21.875rem; //350px
```

Comment:
The height will be ignored and overridden by min-height -- are you sure this is correct?

Reply:
This was done by RW, it was previously set to 0, the 0 is causing the issue it seems, changing it to any value greater than seems to fix it. I just added a value instead of deleting the line due to the "height value needed for IE11".